kimchi/src/verifier.rs

```diff
@@ -212,9 +215,13 @@ where
         //~ 1. Derive $\alpha$ from $\alpha'$ using the endomorphism (TODO: details).
         let alpha = alpha_chal.to_field(endo_r);
 
-        //~ 1. Enforce that the length of the $t$ commitment is of size `PERMUTS`.
-        if self.commitments.t_comm.unshifted.len() != PERMUTS {
-            return Err(VerifyError::IncorrectCommitmentLength("t"));
+        //~ 1. Enforce that the length of the $t$ commitment is of size 7.
+        if self.commitments.t_comm.unshifted.len() != chunk_size * 7 {
```

Comment:
Is this 7 meant to be PERMUTS? Why are we removing this constant? Seemed convenient...

Reply:
See above; it was a misleading coincidence